src/scripts/components/layers/time-slider/time-slider.tsx

```diff
@@ -102,87 +102,89 @@ const TimeSlider: FunctionComponent = () => {
   );
 
   return (
-    <div
-      className={cx(
-        styles.timeSlider,
-        rangeCompare && styles.timeSliderCompare
-      )}>
-      {isPlaying && (
-        <TimePlayback minTime={combined.min} maxTime={combined.max} />
-      )}
-      <div className={styles.container}>
-        <Button
-          className={cx(
-            styles.playButton,
-            rangeCompare && styles.playButtonCompare
-          )}
-          icon={isPlaying ? PauseCircleIcon : PlayCircleIcon}
-          onClick={() => setIsPlaying(!isPlaying)}>
-          {isPlaying ? 'playing' : 'pausing'}
-        </Button>
-        <div className={styles.ranges}>
-          <input
-            className={inputStyles}
-            type="range"
-            value={time}
-            onChange={({target}) => {
-              const newTime = parseInt(target.value, 10);
-              setTime(newTime);
-              debouncedSetGlobeTime(newTime);
-              setIsPlaying(false);
-            }}
-            min={combined.min}
-            max={combined.max}
-            step={stepSize}
-          />
-
-          {rangeMain && (
-            <output
-              className={cx(
-                styles.timeOutput,
-                rangeCompare && styles.timeOutputMain
-              )}
-              style={{
-                left: `${clampedLabelPosition}%`
-              }}>
-              {timeSelectedMain ? format(timeSelectedMain) : false}
-            </output>
-          )}
-
-          {rangeCompare && (
-            <output
-              className={cx(styles.timeOutput, styles.timeOutputCompare)}
-              style={{
-                left: `${clampedLabelPosition}%`
-              }}>
-              {timeSelectedCompare ? format(timeSelectedCompare) : false}
-            </output>
-          )}
-
-          {rangeMain && (
-            <TimeSliderRange
-              range={rangeMain}
-              combined={combined}
-              selectedTimeIndex={timeIndexMain}
+    <div className={styles.sliderBackground}>
```

Comment:
Do we have to add another wrapper div here? Why can't we add the background to the timeslider div?
If we really have to, then the timeSlider div itself doesn't have to have `position: absolute` anymore